docs/docs/user/subscribing.md

```diff
@@ -105,6 +105,19 @@ failure the rescue team has one hour to fix the problem before any event will be
 
 By default inflight TTL is set to 3600 seconds (an hour) and retry backoff is set to 100ms.
 
+## Adaptive backoff
```

Comment:
i think we shouldn't call it adaptive :) it;s just forced backoff, we don't really adapt, just follow the orders